[2024-10-16T18:23:36.077+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-16T18:23:36.103+0000] {__init__.py:51} DEBUG - Loading core task runner: StandardTaskRunner
[2024-10-16T18:23:36.122+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-16T18:23:36.130+0000] {base_task_runner.py:68} DEBUG - Planning to run as the  user
[2024-10-16T18:23:36.134+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.test.dbt_btc_transform.int_hourly_trades manual__2024-10-16T18:23:09.997671+00:00 [queued]> from DB
[2024-10-16T18:23:36.156+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.int_hourly_trades manual__2024-10-16T18:23:09.997671+00:00 [queued]> dependency 'Task Instance State' PASSED: True, Task state queued was valid.
[2024-10-16T18:23:36.162+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.int_hourly_trades manual__2024-10-16T18:23:09.997671+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-16T18:23:36.162+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.int_hourly_trades manual__2024-10-16T18:23:09.997671+00:00 [queued]> dependency 'Task Instance Not Running' PASSED: True, Task is not in running state.
[2024-10-16T18:23:36.175+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.int_hourly_trades manual__2024-10-16T18:23:09.997671+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-16T18:23:36.176+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_dbt_models.test.dbt_btc_transform.int_hourly_trades manual__2024-10-16T18:23:09.997671+00:00 [queued]>
[2024-10-16T18:23:36.176+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.int_hourly_trades manual__2024-10-16T18:23:09.997671+00:00 [queued]> dependency 'Task Concurrency' PASSED: True, Task concurrency is not set.
[2024-10-16T18:23:36.178+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.int_hourly_trades manual__2024-10-16T18:23:09.997671+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-16T18:23:36.186+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.int_hourly_trades manual__2024-10-16T18:23:09.997671+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-16T18:23:36.191+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.int_hourly_trades manual__2024-10-16T18:23:09.997671+00:00 [queued]> dependency 'Pool Slots Available' PASSED: True, There are enough open slots in default_pool to execute the task
[2024-10-16T18:23:36.192+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_dbt_models.test.dbt_btc_transform.int_hourly_trades manual__2024-10-16T18:23:09.997671+00:00 [queued]>
[2024-10-16T18:23:36.192+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-10-16T18:23:36.207+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): test.dbt_btc_transform.int_hourly_trades> on 2024-10-16 18:23:09.997671+00:00
[2024-10-16T18:23:36.211+0000] {standard_task_runner.py:57} INFO - Started process 1661 to run task
[2024-10-16T18:23:36.215+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'run_dbt_models', 'test.dbt_btc_transform.int_hourly_trades', 'manual__2024-10-16T18:23:09.997671+00:00', '--job-id', '97', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt_models.py', '--cfg-path', '/tmp/tmp9cb612f8']
[2024-10-16T18:23:36.215+0000] {standard_task_runner.py:85} INFO - Job 97: Subtask test.dbt_btc_transform.int_hourly_trades
[2024-10-16T18:23:36.217+0000] {cli_action_loggers.py:65} DEBUG - Calling callbacks: [<function default_action_log at 0x7f7ca9505550>]
[2024-10-16T18:23:36.279+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-16T18:23:36.288+0000] {task_command.py:410} INFO - Running <TaskInstance: run_dbt_models.test.dbt_btc_transform.int_hourly_trades manual__2024-10-16T18:23:09.997671+00:00 [running]> on host 888d2c584607
[2024-10-16T18:23:36.288+0000] {settings.py:353} DEBUG - Disposing DB connection pool (PID 1661)
[2024-10-16T18:23:36.289+0000] {settings.py:212} DEBUG - Setting up DB connection pool (PID 1661)
[2024-10-16T18:23:36.289+0000] {settings.py:285} DEBUG - settings.prepare_engine_args(): Using NullPool
[2024-10-16T18:23:36.291+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.test.dbt_btc_transform.int_hourly_trades manual__2024-10-16T18:23:09.997671+00:00 [running]> from DB
[2024-10-16T18:23:36.337+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-16T18:23:36.347+0000] {taskinstance.py:868} DEBUG - Clearing XCom data
[2024-10-16T18:23:36.382+0000] {retries.py:80} DEBUG - Running RenderedTaskInstanceFields._do_delete_old_records with retries. Try 1 of 3
[2024-10-16T18:23:36.399+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='airflow@example.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='run_dbt_models' AIRFLOW_CTX_TASK_ID='test.dbt_btc_transform.int_hourly_trades' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T18:23:09.997671+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-16T18:23:09.997671+00:00'
[2024-10-16T18:23:36.399+0000] {__init__.py:117} DEBUG - Preparing lineage inlets and outlets
[2024-10-16T18:23:36.400+0000] {__init__.py:155} DEBUG - inlets: [], outlets: []
[2024-10-16T18:23:36.400+0000] {bash.py:186} DEBUG - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='airflow@example.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='run_dbt_models' AIRFLOW_CTX_TASK_ID='test.dbt_btc_transform.int_hourly_trades' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T18:23:09.997671+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-16T18:23:09.997671+00:00'
[2024-10-16T18:23:36.401+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-10-16T18:23:36.401+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n            export DBT_SCHEMA=analytics &&\n            cd /opt/airflow/dbt &&\n            /home/airflow/.local/bin/dbt --no-write-json test --target dev --models int_hourly_trades\n            ']
[2024-10-16T18:23:36.411+0000] {subprocess.py:86} INFO - Output:
[2024-10-16T18:23:38.698+0000] {subprocess.py:93} INFO - [0m18:23:38  Running with dbt=1.8.7
[2024-10-16T18:23:39.011+0000] {subprocess.py:93} INFO - [0m18:23:39  Registered adapter: ***=1.8.2
[2024-10-16T18:23:39.402+0000] {subprocess.py:93} INFO - [0m18:23:39  Found 2 models, 18 data tests, 1 source, 540 macros
[2024-10-16T18:23:39.406+0000] {subprocess.py:93} INFO - [0m18:23:39
[2024-10-16T18:23:39.407+0000] {subprocess.py:93} INFO - [0m18:23:39  Nothing to do. Try checking your model configs and model specification args
[2024-10-16T18:23:40.119+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-16T18:23:40.150+0000] {__init__.py:75} DEBUG - Lineage called with inlets: [], outlets: []
[2024-10-16T18:23:40.151+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.test.dbt_btc_transform.int_hourly_trades manual__2024-10-16T18:23:09.997671+00:00 [running]> from DB
[2024-10-16T18:23:40.168+0000] {taskinstance.py:1356} DEBUG - Clearing next_method and next_kwargs.
[2024-10-16T18:23:40.168+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=run_dbt_models, task_id=test.dbt_btc_transform.int_hourly_trades, execution_date=20241016T182309, start_date=20241016T182336, end_date=20241016T182340
[2024-10-16T18:23:40.169+0000] {taskinstance.py:2305} DEBUG - Task Duration set to 3.991994
[2024-10-16T18:23:40.185+0000] {cli_action_loggers.py:83} DEBUG - Calling callbacks: []
[2024-10-16T18:23:40.205+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-10-16T18:23:40.269+0000] {dagrun.py:697} DEBUG - number of tis tasks for <DagRun run_dbt_models @ 2024-10-16 18:23:09.997671+00:00: manual__2024-10-16T18:23:09.997671+00:00, state:running, queued_at: 2024-10-16 18:23:10.005456+00:00. externally triggered: True>: 5 task(s)
[2024-10-16T18:23:40.269+0000] {dagrun.py:718} DEBUG - number of scheduleable tasks for <DagRun run_dbt_models @ 2024-10-16 18:23:09.997671+00:00: manual__2024-10-16T18:23:09.997671+00:00, state:running, queued_at: 2024-10-16 18:23:10.005456+00:00. externally triggered: True>: 1 task(s)
[2024-10-16T18:23:40.272+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-16T18:23:09.997671+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-16T18:23:40.273+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-16T18:23:09.997671+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-16T18:23:40.274+0000] {taskinstance.py:1103} DEBUG - Dependencies all met for dep_context=None ti=<TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-16T18:23:09.997671+00:00 [None]>
[2024-10-16T18:23:40.278+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
